apps/ledger-live-desktop/tests/models/Transaction.ts

```diff
@@ -1,10 +1,24 @@
 import { Account } from "../enum/Account";
+import { Token } from "../enum/Token";
 
-export class Transaction {
+export abstract class Transaction {
   constructor(
     public readonly accountToDebit: Account,
-    public readonly recipient: string,
+    public readonly accountToCredit: Account,
     public readonly amount: string,
     public readonly speed: string,
   ) {}
 }
+export class BasicTransaction extends Transaction {}
+export class TokenTransaction extends Transaction {
+  constructor(
+    accountToDebit: Account,
+    public readonly accountToCredit1: Account,
+    public readonly accountToCredit2: Account,
+    amount: string,
+    speed: string,
```

Comment:
speed could use [Fee](https://github.com/LedgerHQ/ledger-live/pull/7200/files#diff-d9b544cb5a433b07081b9f7427cb8bc8d2e2643935006dd585fdcea4b5c60a7bR1) enum isn't it ?